docs/pages/api/access-plugin.mdx

```diff
@@ -0,0 +1,738 @@
+
+### Create a row
+
+Below `makeRowData`, add the following:
+
+```go
+func (g *googleSheetsPlugin) createRow(ar types.AccessRequest) error {
+
+	row := g.makeRowData(ar)
+
+	req := sheets.BatchUpdateSpreadsheetRequest{
+		Requests: []*sheets.Request{
+			{
+				AppendCells: &sheets.AppendCellsRequest{
+
+					Fields: "*",
+					Rows: []*sheets.RowData{
+						row,
+					},
+				},
+			},
+		},
+	}
+
+	resp, err := g.sheetsClient.BatchUpdate(spreadSheetID, &req).Do()
+
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	if resp.HTTPStatusCode == 201 || resp.HTTPStatusCode == 200 {
+		fmt.Println("Successfully created a row")
+	} else {
+		fmt.Printf(
+			"Unexpected response code creating a row: %v\n",
+			resp.HTTPStatusCode,
+		)
+	}
+
+	return nil
+
+}
+```
+
+`createRow` submits a request to the Google Sheets API to create a new row based
+on an incoming Access Request, using the data returned by `makeRowData`. It
+returns an error if the attempt to create a row failed. 
+
+It assembles a `sheets.BatchUpdateSpreadsheetRequest` and sends it to the Google
+Sheets API using `g.sheetsClient.BatchUpdate()`, returning errors encountered
+while sending the request.
+
+We log unexpected HTTP status codes without returning an error since these may
+be transient server-side issues. A production Access Request plugin would handle
+these situations in a more sophisticated way, e.g., storing the request so it
+can retry it later.
+
+### Update a row
+
+The code for updating a row is similar to the code for creating a new row. Add
+this function below `createRow`:
+
+```go
+func (g *googleSheetsPlugin) updateRow(ar types.AccessRequest, rowNum int64) error {
+
+	row := g.makeRowData(ar)
+
+	req := sheets.BatchUpdateSpreadsheetRequest{
+		Requests: []*sheets.Request{
+			{
+				UpdateCells: &sheets.UpdateCellsRequest{
+
+					Fields: "*",
+					Start: &sheets.GridCoordinate{
+						RowIndex: rowNum,
+					},
+					Rows: []*sheets.RowData{
+						row,
+					},
+				},
+			},
+		},
+	}
+	resp, err := g.sheetsClient.BatchUpdate(spreadSheetID, &req).Do()
+
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	if resp.HTTPStatusCode == 201 || resp.HTTPStatusCode == 200 {
+		fmt.Println("Successfully updated a row")
+	} else {
+		fmt.Printf(
+			"Unexpected response code updating a row: %v\n",
+			resp.HTTPStatusCode,
+		)
+	}
+
+	return nil
+
+}
+```
+
+The only difference between `updateRow` and `createRow` is that we send a
+`&sheets.UpdateCellsRequest` instead of a `&sheets.AppendCellsRequest`. This
+function takes the number of a row within the spreadsheet to update and sends a
+request to update that row with information from the provided Access Request.
+
+### Determine where to update the spreadsheet
+
+Below `updateRow`, add this function:
+
+```go
+func (g *googleSheetsPlugin) updateSpreadsheet(ar types.AccessRequest) error {
+
+	s, err := g.sheetsClient.Get(spreadSheetID).IncludeGridData(true).Do()
+
+	if err != nil {
+		return trace.Wrap(err)
```

Comment:
Either use trace consistently, or don't use it. A reader unfamiliar with Go will be wondering why we're not consistent here.